common/start.S

```diff
@@ -1,14 +1,18 @@
+#include "config.h"
+
     .section .text.entry
     .type _entry, @function
     .global _entry
 _entry:
+#if defined(ESP32C3)
```

Comment:
I can't figure out, why is it only necessary to initialize the .data section on C3? Where is the .data section initialized on the H2?